Gems/HttpRequestor/Code/Source/HttpRequestManager.cpp

```diff
@@ -49,7 +49,7 @@ namespace HttpRequestor
         {
             m_thread.join();
         }
-
+        AWSNativeSDKInit::InitializationManager::Shutdown();
```

Comment:
This line needs to be deleted.
@burelc-amzn already added this line above a couple of days ago.

Reply:
When and where was it added?

Reply:
Look upwards 7 lines